app/src/main/java/org/mozilla/fenix/tabhistory/TabHistoryView.kt

```diff
@@ -60,6 +57,7 @@ class TabHistoryView(
     init {
         tabHistoryRecyclerView.adapter = adapter
         tabHistoryRecyclerView.layoutManager = layoutManager
+        tabHistoryRecyclerView.itemAnimator = null
```

Comment:
Is this just to disable change animations? We could do something like this:

```suggestion
        (tabHistoryRecyclerView.itemAnimator as SimpleItemAnimator).supportsChangeAnimations = false
```

We also do this in `HistoryView` since the alpha animation in the change animations causes flickering.

Reply:
Did a little bit of testing and I'm not sure we need to disable any animations at all since `ListAdapter` won't rebind anything if it didn't change.